--- /dev/fd/63	2026-01-14 18:00:21
+++ /dev/fd/62	2026-01-14 18:00:21
@@ -1,8 +1,12 @@
 
---------+----------------------------------------------+----------+-------
+--------+---------------------------------------------+----------+-------
 Name
 app_blockedmembernode
 app_blockedmembernode_id_seq
+app_chain
+app_chain_id_seq
+app_chainmember
+app_chainmember_id_seq
 app_event
 app_event_id_seq
 app_eventlog
@@ -13,6 +17,10 @@
 app_ipaddress_id_seq
 app_localreplica
 app_localreplica_id_seq
+app_mediatype
+app_mediatype_id_seq
+app_mediatypeproperty
+app_mediatypeproperty_id_seq
 app_node
 app_node_id_seq
 app_permission
@@ -23,22 +31,24 @@
 app_remotereplica_id_seq
 app_replicainfo
 app_replicainfo_id_seq
-app_replicaobsolescencechainreference
-app_replicaobsolescencechainreference_id_seq
+app_replicarevisionchainreference
+app_replicarevisionchainreference_id_seq
 app_replicastatus
 app_replicastatus_id_seq
 app_replicationpolicy
 app_replicationpolicy_id_seq
 app_replicationqueue
 app_replicationqueue_id_seq
+app_resourcemap
+app_resourcemap_id_seq
+app_resourcemapmember
+app_resourcemapmember_id_seq
 app_scienceobject
 app_scienceobject_id_seq
 app_scienceobjectchecksumalgorithm
 app_scienceobjectchecksumalgorithm_id_seq
 app_scienceobjectformat
 app_scienceobjectformat_id_seq
-app_seriesidtoscienceobject
-app_seriesidtoscienceobject_id_seq
 app_subject
 app_subject_id_seq
 app_systemmetadatarefreshqueue
@@ -49,6 +59,20 @@
 app_useragent_id_seq
 app_whitelistforcreateupdatedelete
 app_whitelistforcreateupdatedelete_id_seq
+auth_group
+auth_group_id_seq
+auth_group_permissions
+auth_group_permissions_id_seq
+auth_permission
+auth_permission_id_seq
+auth_user
+auth_user_groups
+auth_user_groups_id_seq
+auth_user_id_seq
+auth_user_user_permissions
+auth_user_user_permissions_id_seq
+django_content_type
+django_content_type_id_seq
 django_migrations
 django_migrations_id_seq
 relations
